manifest-build-tools/lib/manifest.py

```diff
@@ -12,7 +12,7 @@
 
 from gitbits import GitBit
 
-
+manifest_sample = "manifest.json"
```

Comment:
still perfer it with better naming :-)   like manifest.json.sample or manifest.json.template